### PR TITLE
feat(web-vanguard): implemented the hero section on teams page

### PR DESCRIPTION
**Pull request by Yared04**
Changed the branch name of for this pr #21.

### CODE REVIEW

**Comment by YiseBoge** [comment]
normally no need to create new pr for renaming, but since i have not done anything on the other branch, we can continue here
make sure you add the sceenshot and explanations on the description tho

also please edit the PR name to follow the conventional commit rules as it is what is used by default with squash and merge

File: Web/starter_project_vanguard/styles/globals.css [hunk 1]
```diff
@@ -21,7 +21,7 @@ body {
   background: linear-gradient(
       to bottom,
       transparent,
-      rgb(var(--background-end-rgb))
+      rgb(var(--background-end-rgb))
```
Comment (YiseBoge) [inline Web/starter_project_vanguard/styles/globals.css:24]: please remove the extra space here 
let's not track changes that don't involve your feature

File: Web/starter_project_vanguard/public/Vector (1).png
Comment (YiseBoge) [inline Web/starter_project_vanguard/public/Vector (1).png:1]: this is not a good name for the file, make it descriptive

this is also not the right folder to put all these images, i would at least expect them to be inside `img/[your-feature]`

File: Web/starter_project_vanguard/pages/index.tsx
Comment (YiseBoge) [inline Web/starter_project_vanguard/pages/index.tsx:1]: similar to the other PR, this is not a change that involves your PR, please bring back the page

File: Web/starter_project_vanguard/package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.3",
     "@types/node": "18.15.11",
-    "@types/react": "18.0.35",
+    "@types/react": "^18.0.35",
```
Comment (YiseBoge) [inline Web/starter_project_vanguard/package.json:20]: why is this?

File: Web/starter_project_vanguard/components/team/HeroSection.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+const HeroSection = () => {
+  return (
+    <div className="flex mx-10">
+        <div className="basis-2/5 ">
+            <h1 className="uppercase font-bold text-5xl text-primary-text flex flex-col gap-4"><span>The team we're</span>  <span>currently</span> <span>working with</span> </h1>
+            <p className="text-secondary-text mt-8 text-xl">Meet our development team is a small but highly skilled and experienced group of professionals. We have a talented mix of web developers, software engineers, project managers and quality assurance specialists who are passionate about developing exceptional products and services.</p>
+        </div>
+        <div className=" relative basis-3/5 p-40 flex flex-col gap-4 bg-center bg-no-repeat bg-contain text-primary-text" style={{backgroundImage: 'url("vector.png")'}}>
+          <Image className=" md:absolute right-24 -top-6" width={230} height={250} src={'/vector (1).png'} alt={"hero image"}/>
```
Comment (YiseBoge) [inline Web/starter_project_vanguard/components/team/HeroSection.tsx:11]: better to do this in a loop - you can have a list of image url's with their class names and loop through it

File: Web/starter_project_vanguard/components/team/HeroSection.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+          <Image className="absolute -top-8 left-14 " width={230} height={250} src={'/vector (2).png'} alt={"hero image"}/>
+          <h1 className="uppercase font-bold text-5xl ml-24"><span className=" text-[#264FAD]">Team</span>  work</h1>
+          <h1 className="uppercase font-bold text-5xl ml-24">collaboration</h1>
+          <h1 className="uppercase font-bold text-5xl mx-24 "><span className=" text-[#264FAD]">Hard</span> work</h1>
```
Comment (YiseBoge) [inline Web/starter_project_vanguard/components/team/HeroSection.tsx:15]: would be nice if there is a way to globally define colors and reuse them with other people too

File: Web/starter_project_vanguard/components/team/HeroSection.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import Image from "next/image"
+
+const HeroSection = () => {
```
Comment (YiseBoge) [inline Web/starter_project_vanguard/components/team/HeroSection.tsx:3]: Please define a component with a syntac like React.FC (or whatever is the best way for nextJS) to clarify what is what

File: Web/starter_project_vanguard/components/team/Member.tsx
Comment (YiseBoge) [inline Web/starter_project_vanguard/components/team/Member.tsx:1]: i was thinking i have done the review to this file before but now i realise this is also a file that has nothing to do with the feature of this PR
let's not have such changes

File: Web/starter_project_vanguard/next.config.js [hunk 6]
```diff
@@ -1,6 +1,17 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  images: {
```
Comment (YiseBoge) [inline Web/starter_project_vanguard/next.config.js:4]: same comment that i had on your other PR here to

**Review by YiseBoge — CHANGES_REQUESTED** [review]
I have left some comments but i have to admit it gets confusing to track multiple PR's with changes that are common in between them (you also have changes that are also done in the team members PR)

File: Web/starter_project_vanguard/next.config.js (hunk 6 above)
Comment (Yared04) [reply Web/starter_project_vanguard/next.config.js:5]: Done

**Comment by Yared04** [comment]
hero

**Review by YiseBoge — APPROVED** [review]
Looks good
do some of the renamings we talked about in the other PR

**Pull request merged**